src/content/en/2020/media.md

```diff
@@ -23,3 +24,439 @@ queries: 05_Media
 #featured_stat_label_3: TODO
```

Comment:
@tpiros can you suggest a featured quote (typically a few sentences) and 3 stats with labels to appear on the home page to promote this chapter.

This is what last years chapter had:

![2019 featured quote](https://user-images.githubusercontent.com/10931297/104482935-7382d280-55bf-11eb-9344-1f6703c781c1.png)

Reply:
I'd suggest the following for the feat quote (Ben's image intro, modified slightly):

```
Images and videos offer us the potential for a most powerful pairing: instant communication, capable of triggering an innate emotional response. However, they require considered implementation techniques to avoid also being a burden to our web pages.
```

In terms of stats I'd add (if possible to find) **median size of home page with video**, **% of all sites using an image** and **most used image format**. @bazzadp would you be able to help us find this data, or is there no capacity to do this atm?

Reply:
Looks like we don't have any media sizes data. That seems a bit of a miss to me to be honest! Though I guess covered in the page weight chapter. Maybe should add a paragraph and link to that?

Anyway what do you think about the following for featured quote and stats?

```
featured_quote: Images and videos offer us the potential for a most powerful pairing&colon; instant communication, capable of triggering an innate emotional response. However, they require considered implementation techniques to avoid also being a burden to our web pages.
featured_stat_1: 84.64%
featured_stat_label_1: Usage of WebP in mobile <code>&lt;Picture&gt;</code> element
featured_stat_2: 40.26%
featured_stat_label_2: JPG images as a percentage of all images
featured_stat_3: 57.22%
featured_stat_label_3: Video elements on desktop with the autoplay attribute
```

It looks like this on the home page:

![Featured Quote](https://user-images.githubusercontent.com/10931297/104728302-9c75a580-572e-11eb-8188-6e796f624c25.png)